src/main/archetype/ui.tests/test-module/lib/util/forms.selectors.js

```diff
@@ -0,0 +1,47 @@
+var selectors = {
+    editor : {
+        layerSwitcher : '.editor-GlobalBar-layerSwitcher',
+        layerSelector : {
+            initial : '.editor-GlobalBar-layerSwitcherPopoverContent button[data-layer="initial"]',
+            structure : '.editor-GlobalBar-layerSwitcherPopoverContent button[data-layer="structure"]'
+        },
+        previewButtonSelector : '.editor-GlobalBar-previewTrigger',
+        previewLayerSelector : '.aem-AuthorLayer-Preview',
+        currentLayerButtonSelector : '.editor-GlobalBar-layerCurrent',
+        editorLayerSelector : 'html.aem-AuthorLayer-Theme.Edit',
+        editToolBar : {
+            configure : '#EditableToolbar button[data-action="CONFIGURE"]'
+        },
+        sidePanelToggleButton : '.toggle-sidepanel.editor-GlobalBar-item',
+        sidePanelOpen : '#SidePanel.sidepanel-opened',
+        sidePanel : {
+            afPropertiesContainer : '#sidepanel-guide-properties',
+            stylePropertySheetSelector : '#style-propertySheetContainer',
+            propertiesTab : 'coral-tab[title="Properties"]'
+        },
+        overlay : {
+            guideFieldLabelOvelaySelector : '#af-overlay-selector-text > button[data-target-selector="af_fieldlabel"]'
+        },
+        themeEditor : {
+            themeEditorObjSelector : '.themeEditableObject'
+        },
+        templateEditor : {
+            structureLayer : {
+                policyButton : 'button.cq-editable-action[title="Policy"]',
+                policyPage : {
+                    allowedComponent : {
+                        adaptiveFormGroup : '.js-cq-AllowedComponents-groupCheckbox[value="group:Adaptive Form"]'
+                    },
+                    cancel : 'button.cq-dialog-header-action[title="Cancel"]'
+                }
+            }
+        }
+    },
+    content : {
```

Comment:
Can we avoid using these selectors for now, since ".guideFieldNode" is not documented ? For now, let's not verify the form HTML

Reply:
Using .guidefield now. Form html verification is required for checking AF field presence (count) in AF template.